config/locales/en.yml

```diff
@@ -156,3 +156,228 @@ en:
   language_names:
     en: English
     cy: Cymraeg
+  content_item:
+    schema_name:
+      # `national` & `official` are old document types for
```

Comment:
This comment relates to a PR that was merged 3 years ago! I think we can safely leave out those translations?